src/tooling/piral-cli-parcel/README.md

```diff
@@ -1,3 +1,7 @@
+---
```

Comment:
Modifying this MD is not intended;

- it will be inconsistent to the other READMEs
- it will potentially misrender on GitHub / NPM / etc.

Same applies to the `piral-cli-webpack` README

Reply:
removed the meta tag 'title'